### DISCUSSION TITLE
How to use bundled library in Deno?

### DISCUSSION THREAD

**Discussion by krutoo** [KingSora/OverlayScrollbars#621] [Q&A] [answered]
Hi, i am trying to use OverlayScrollbars in Deno, in my case i forced to bundle server side app by webpack and keep libraries from node_modules in bundle

In source code of this library there is:

```js
const n = t ? window : {};
// ...
const v = t && Node.ELEMENT_NODE;

```

Deno has global variable "window".

When I try to run bundle by `deno run bundle.js` I have 

```
ReferenceError: Node is not defined
```

How can i solve this? I have to bundle server code. 

Maybe we can make checks lazy by wrap it into functions? Or maybe change conditions

**Comment by KingSora** [comment, marked as answer]
Good day @krutoo :)

This seems to be a quirk in `Deno`, because there the `window` global is defined. This is not the case in `Node` or `Bun` which is why it doesn't happen in those runtimes.

I've published a new version [`v2.7.0`](https://www.npmjs.com/package/overlayscrollbars/v/2.7.0) which includes an improvement for this issue.
